src/pages/Home/Home.page.jsx

```diff
@@ -1,37 +1,17 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React from 'react';
+import VideoGrid from '../../components/VideoGrid';
+import mockedData from '../../mockData/mockData.json';
+import Styled from './styled';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+const { items } = mockedData;
 
 function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
-
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
-
   return (
-    <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
+    <section>
+      <div>
```

Comment:
I do not see the need for a `div` here since you are wrapping everything with a `section`